examples/examples_test.go

```diff
@@ -81,17 +78,6 @@ func TestExamples(t *testing.T) {
 			Dir:          path.Join(cwd, "dynamic-provider/multiple-turns-2"),
 			Dependencies: []string{"@pulumi/pulumi"},
 		},
-		{
-			Dir:          path.Join(cwd, "compat/v0.10.0/minimal"),
```

Comment:
Why did we drop this test?  Can't we replace with a version of "minimal" that uses the new "overrides"?

Reply:
The test only works on 6.10.X because that's the only version of node 0.10.0 supported. It was passing everywhere because it incorrectly linked in the version of the just built `@pulumi/pulumi` package.

I changed things up such that we will run the test on 6.10.X, but on other nodes, we just skip it.